packages/medusa/src/api/routes/admin/notes/index.js

```diff
@@ -0,0 +1,23 @@
+import { Router } from "express"
+import middlewares from "../../../middlewares"
+
+const route = Router()
+
+export default app => {
+  app.use("/notes", route)
+
+  route.get("/:id", middlewares.wrap(require("./get-note").default))
+
+  route.get(
+    "/resource/:resource_id",
+    middlewares.wrap(require("./list-notes").default)
+  )
+
```

Comment:
Maybe consider removing this option and instead creating an endpoint within orders that would allow you to do this: `GET /admin/orders/:id/notes`

Let me know what you think about this? Please pitch in too @olivermrbl

Reply:
I think - as the resource may be different than the order - it should be kept separate. Even though this is the only place in which we use it currently.

Reply:
Yes agree - however, I am not very happy about the path here. `GET /admin/notes/resource/id` seems weird.

Maybe we should just stick with `GET /admin/notes?resource_id=id` - I think this is more RESTy